tests/unit/test_project.py

```diff
@@ -327,14 +327,12 @@ def test_project_load(yaml_data, yaml_loaded_data, tmp_path):
             attr = "rock_license"
         if attr == "platforms":
             # platforms get mutated at validation time
-            assert project.__getattribute__(attr).keys() == v.keys()
+            assert getattr(project, attr).keys() == v.keys()
```

Comment:
can this be separate?

Reply:
Ah, this shouldn't be in this PR at all, it was done locally to be able to run linters. Removing.